tasks/release.py

```diff
@@ -1109,15 +1160,39 @@ def create_rc(ctx, major_versions="6,7", patch_version=False, upstream="origin")
             code=1,
         )
 
+    create_release_pr(
+        "[release] Update release.json and Go modules for {versions_string}",
```

Comment:
```suggestion
        f"[release] Update release.json and Go modules for {versions_string}",
```
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Typo